src/registrar/fixtures.py

```diff
@@ -73,7 +99,58 @@ def load(cls):
                 logger.debug("User object created for %s" % admin["first_name"])
             except Exception as e:
                 logger.warning(e)
-        logger.debug("All users loaded.")
+        logger.debug("All superusers loaded.")
+
+        logger.info("Going to load %s CISA analysts (staff)" % str(len(cls.STAFF)))
+        for staff in cls.STAFF:
+            try:
+                user, _ = User.objects.get_or_create(
+                    username=staff["username"],
+                )
+                user.is_superuser = False
+                user.first_name = staff["first_name"]
+                user.last_name = staff["last_name"]
+                user.is_staff = True
+                user.is_active = True
+
+                for permission in cls.STAFF_PERMISSIONS:
+                    app_label = permission["app_label"]
+                    model_name = permission["model"]
+                    permissions = permission["permissions"]
+
+                    # Retrieve the content type for the app and model
+                    content_type = ContentType.objects.get(
```

Comment:
I believe you said accessing the contenttype was throwing an error on fixtures on the first couple loads, and thus when (if) we fix the fixture bug it won't work correctly. Is this the line you were referring to? (I am not seeing the same error in the console log though).

Reply:
Looks like it to me

![Screen Shot 2023-07-06 at 10 39 28 AM](https://github.com/cisagov/getgov/assets/107004823/bec1023d-e004-4610-8b50-a4edc8f2c7fe)